app/assets/stylesheets/projects.css.scss

```diff
@@ -27,3 +27,21 @@ h5 span.favorite {
 .project_links {
   margin-bottom: 6%;
 }
+
+.tag_list {
+  border-radius: 5px;
+  color: white;
+  padding: 10px;
```

Comment:
Properties should be ordered border-radius, color, margin-bottom, overflow-x, overflow-y, padding, white-space